rules-placeholder/windows/win_certificate_exported.yml

```diff
@@ -0,0 +1,33 @@
+title: Certificate Exported
+id: e2b5163d-7deb-4566-9af3-40afea6858c3
+status: experimental
+description: Detects when a user or mimikatz exports a certificate (and possibly the private key as well) from the local Windows certificate store.
+references:
+    - https://www.splunk.com/en_us/blog/security/breaking-the-chain-defending-against-certificate-services-abuse.html
+author: Zach Mathis
+date: 2023/05/13
+tags:
+    - attack.credential_access
+    - attack.t1649
+logsource:
+    product: windows
+    definition: 'The CAPI2/Operational log needs to be enabled as it is disabled by default. Powershell ScriptBlock logging should also be enabled.'
+detection:
+    select_CAPI_log:
+        Channel: Microsoft-Windows-CAPI2/Operational
+        EventID: 70 # "Acquire Certificate Private Key"
+    select_CS_client_log:
+        Channel: Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational
+        EventID: 1007 # "A certificate has been exported"
+    select_PwSh_log:
```

Comment:
Can be an update to the rule [aa7a3fce-bef5-4311-9cc1-5f04bb8c308c](https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml)
category: ps_script cover the channel and EventID4104
detection is allready for Export-PfxCertificate

Reply:
Adding to the [aa7a3fce-bef5-4311-9cc1-5f04bb8c308c](https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml) rule for powershell export-certificate sounds good to me. I removed that part of the rule.